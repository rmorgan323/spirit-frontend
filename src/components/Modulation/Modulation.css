.Modulation .category-content {
  display: flex;
  margin: auto;
  width: fit-content; }
