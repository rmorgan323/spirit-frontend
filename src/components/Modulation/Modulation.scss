.Modulation {

  .category-content {
    display: flex;
  }
}