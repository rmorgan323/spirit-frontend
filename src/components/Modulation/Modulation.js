import React from 'react';
import SessionHeader from '../SessionHeader/SessionHeader';
import CategorySliders from '../CategorySliders/CategorySliders';
import { modulation } from '../../language/language';
import './Modulation.css';

const Modulation = () => {
  const { header, columns } = modulation;

  return (
    <div className="Modulation">
      <SessionHeader
        title={header.title}
        headings={header.terms}
        databaseNamesArray={header.db}
      />

      <div className="category-content">
        {Object.keys(columns).map(column => {
          const { title, terms, db, reminderPlacement, reminder } = columns[column];

          return (
            <CategorySliders
              key={db}
              title={title}
              slidersArray={terms}
              databaseNamesArray={db}
              reminderAsterisk={reminderPlacement && reminderPlacement}
              reminder={reminder}
            />
          );
        })}
      </div>
    </div>
  );
};

export default Modulation;
