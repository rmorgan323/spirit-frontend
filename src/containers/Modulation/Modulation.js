import React from 'react';
import SessionHeader from '../SessionHeader/SessionHeader';
import CategorySliders from '../../components/CategorySliders/CategorySliders';
import './Modulation.css';

const Modulation = () => {
  
  return (
    <div className="Modulation">
      <SessionHeader 
        title={'Modulation Capacities: Sensory Modulation to Self-Regulation'}
        headings={['Vestibular', 'Proprioception', 'Tactile', 'Auditory', 'Visual', 'Intero']}
        databaseNamesArray={['mod_h_vestibular', 'mod_h_proprioception', 'mod_h_tactile', 'mod_h_auditory', 'mod_h_visual', 'mod_h_intero']}
      />
      <div className="category-content">
        <CategorySliders 
          slidersArray={['Defensiveness', 'Over', 'Under', 'Variable', 'Arousability']}
          databaseNamesArray={['mod_1_defensiveness', 'mod_1_over', 'mod_1_under', 'mod_1_variable', 'mod_1_arousability']}
          reminderAsterisk={[null, '*', '*', '*', '*']}
          reminder={'* If selected, remember to make a hypothesized arousal chart.'}
        />
        <CategorySliders 
          title={'Capacities'}
          slidersArray={['Attention', 'Action', 'Affect', 'Autonomic']}
          databaseNamesArray={['mod_2_attention', 'mod_2_action', 'mod_2_affect', 'mod_2_autonomic']}
        />
        <CategorySliders 
          title={'Self-Regulation'}
          slidersArray={['Physiological', 'Sensory', 'Motor', 'Emotional', 'Cognitive', 'Social', 'Prosocial']}
          databaseNamesArray={['mod_3_physiological', 'mod_3_sensory', 'mod_3_motor', 'mod_3_emotional', 'mod_3_cognitive', 'mod_3_social', 'mod_3_prosocial']}
        />
        <CategorySliders 
          title={'Regulatory Capacities'}
          slidersArray={['Passive', 'Co-Regulation', 'Approach', 'Adaptive']}
          databaseNamesArray={['mod_4a_passive', 'mod_4a_co_regulation', 'mod_4a_approach', 'mod_4a_adaptive']}
        />
        <CategorySliders 
          slidersArray={['Active', 'Self-Regulation', 'Avoid', 'Maladaptive']}
          databaseNamesArray={['mod_4b_active', 'mod_4b_self_regulation', 'mod_4b_avoid', 'mod_4b_maladaptive']}
        />
      </div>
    </div>
  )
}

export default Modulation;