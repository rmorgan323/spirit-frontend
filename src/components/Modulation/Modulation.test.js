import React from 'react';
import Modulation from './Modulation';
import { shallow } from 'enzyme';

describe('Modulation tests', () => {
  it('Should match snapshot', () => {
    const renderedApp = shallow(<Modulation />);

    expect(renderedApp).toMatchSnapshot();
  });
});
