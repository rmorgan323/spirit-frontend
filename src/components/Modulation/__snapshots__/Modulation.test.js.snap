// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modulation tests Should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Modulation />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <SessionHeader
          databaseNamesArray={
            Array [
              "mod_h_vestibular",
              "mod_h_proprioception",
              "mod_h_tactile",
              "mod_h_auditory",
              "mod_h_visual",
              "mod_h_intero",
            ]
          }
          headings={
            Array [
              "Vestibular",
              "Proprioception",
              "Tactile",
              "Auditory",
              "Visual",
              "Intero",
            ]
          }
          title="Modulation Capacities: Sensory Modulation to Self-Regulation"
        />,
        <div
          className="category-content"
        >
          <CategorySliders
            databaseNamesArray={
              Array [
                "mod_1_defensiveness",
                "mod_1_over",
                "mod_1_under",
                "mod_1_variable",
              ]
            }
            reminder="* If selected, remember to make a hypothesized arousal chart"
            reminderAsterisk={
              Array [
                null,
                "*",
                "*",
                "*",
                "*",
              ]
            }
            slidersArray={
              Array [
                "Defensiveness",
                "Over",
                "Under",
                "Variable",
              ]
            }
            title="Capacities"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "mod_2_attention",
                "mod_2_action",
                "mod_2_affect",
                "mod_2_autonomic",
                "mod_2_arousability",
              ]
            }
            reminder={undefined}
            reminderAsterisk={
              Array [
                null,
                null,
                null,
                null,
                "*",
              ]
            }
            slidersArray={
              Array [
                "Attention",
                "Action",
                "Affect",
                "Autonomic",
                "Arousability",
              ]
            }
            title="Capacities"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "mod_3_physiological",
                "mod_3_sensory",
                "mod_3_motor",
                "mod_3_emotional",
                "mod_3_cognitive",
                "mod_3_social",
                "mod_3_prosocial",
              ]
            }
            reminder={undefined}
            reminderAsterisk={undefined}
            slidersArray={
              Array [
                "Physiological",
                "Sensory",
                "Motor",
                "Emotional",
                "Cognitive",
                "Social",
                "Prosocial",
              ]
            }
            title="Self-Regulation"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "mod_4a_passive",
                "mod_4a_co_regulation",
                "mod_4a_approach",
                "mod_4a_adaptive",
              ]
            }
            reminder={undefined}
            reminderAsterisk={undefined}
            slidersArray={
              Array [
                "Passive",
                "Co-Regulation",
                "Approach",
                "Adaptive",
              ]
            }
            title="Regulatory Capacities"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "mod_4b_active",
                "mod_4b_self_regulation",
                "mod_4b_avoid",
                "mod_4b_maladaptive",
              ]
            }
            reminder={undefined}
            reminderAsterisk={undefined}
            slidersArray={
              Array [
                "Active",
                "Self-Regulation",
                "Avoid",
                "Maladaptive",
              ]
            }
            title="Regulatory Capacities"
          />
        </div>,
      ],
      "className": "Modulation",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "databaseNamesArray": Array [
            "mod_h_vestibular",
            "mod_h_proprioception",
            "mod_h_tactile",
            "mod_h_auditory",
            "mod_h_visual",
            "mod_h_intero",
          ],
          "headings": Array [
            "Vestibular",
            "Proprioception",
            "Tactile",
            "Auditory",
            "Visual",
            "Intero",
          ],
          "title": "Modulation Capacities: Sensory Modulation to Self-Regulation",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_1_defensiveness",
                  "mod_1_over",
                  "mod_1_under",
                  "mod_1_variable",
                ]
              }
              reminder="* If selected, remember to make a hypothesized arousal chart"
              reminderAsterisk={
                Array [
                  null,
                  "*",
                  "*",
                  "*",
                  "*",
                ]
              }
              slidersArray={
                Array [
                  "Defensiveness",
                  "Over",
                  "Under",
                  "Variable",
                ]
              }
              title="Capacities"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_2_attention",
                  "mod_2_action",
                  "mod_2_affect",
                  "mod_2_autonomic",
                  "mod_2_arousability",
                ]
              }
              reminder={undefined}
              reminderAsterisk={
                Array [
                  null,
                  null,
                  null,
                  null,
                  "*",
                ]
              }
              slidersArray={
                Array [
                  "Attention",
                  "Action",
                  "Affect",
                  "Autonomic",
                  "Arousability",
                ]
              }
              title="Capacities"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_3_physiological",
                  "mod_3_sensory",
                  "mod_3_motor",
                  "mod_3_emotional",
                  "mod_3_cognitive",
                  "mod_3_social",
                  "mod_3_prosocial",
                ]
              }
              reminder={undefined}
              reminderAsterisk={undefined}
              slidersArray={
                Array [
                  "Physiological",
                  "Sensory",
                  "Motor",
                  "Emotional",
                  "Cognitive",
                  "Social",
                  "Prosocial",
                ]
              }
              title="Self-Regulation"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_4a_passive",
                  "mod_4a_co_regulation",
                  "mod_4a_approach",
                  "mod_4a_adaptive",
                ]
              }
              reminder={undefined}
              reminderAsterisk={undefined}
              slidersArray={
                Array [
                  "Passive",
                  "Co-Regulation",
                  "Approach",
                  "Adaptive",
                ]
              }
              title="Regulatory Capacities"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_4b_active",
                  "mod_4b_self_regulation",
                  "mod_4b_avoid",
                  "mod_4b_maladaptive",
                ]
              }
              reminder={undefined}
              reminderAsterisk={undefined}
              slidersArray={
                Array [
                  "Active",
                  "Self-Regulation",
                  "Avoid",
                  "Maladaptive",
                ]
              }
              title="Regulatory Capacities"
            />,
          ],
          "className": "category-content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "mod_1_defensiveness,mod_1_over,mod_1_under,mod_1_variable",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "mod_1_defensiveness",
                "mod_1_over",
                "mod_1_under",
                "mod_1_variable",
              ],
              "reminder": "* If selected, remember to make a hypothesized arousal chart",
              "reminderAsterisk": Array [
                null,
                "*",
                "*",
                "*",
                "*",
              ],
              "slidersArray": Array [
                "Defensiveness",
                "Over",
                "Under",
                "Variable",
              ],
              "title": "Capacities",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "mod_2_attention,mod_2_action,mod_2_affect,mod_2_autonomic,mod_2_arousability",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "mod_2_attention",
                "mod_2_action",
                "mod_2_affect",
                "mod_2_autonomic",
                "mod_2_arousability",
              ],
              "reminder": undefined,
              "reminderAsterisk": Array [
                null,
                null,
                null,
                null,
                "*",
              ],
              "slidersArray": Array [
                "Attention",
                "Action",
                "Affect",
                "Autonomic",
                "Arousability",
              ],
              "title": "Capacities",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "mod_3_physiological,mod_3_sensory,mod_3_motor,mod_3_emotional,mod_3_cognitive,mod_3_social,mod_3_prosocial",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "mod_3_physiological",
                "mod_3_sensory",
                "mod_3_motor",
                "mod_3_emotional",
                "mod_3_cognitive",
                "mod_3_social",
                "mod_3_prosocial",
              ],
              "reminder": undefined,
              "reminderAsterisk": undefined,
              "slidersArray": Array [
                "Physiological",
                "Sensory",
                "Motor",
                "Emotional",
                "Cognitive",
                "Social",
                "Prosocial",
              ],
              "title": "Self-Regulation",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "mod_4a_passive,mod_4a_co_regulation,mod_4a_approach,mod_4a_adaptive",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "mod_4a_passive",
                "mod_4a_co_regulation",
                "mod_4a_approach",
                "mod_4a_adaptive",
              ],
              "reminder": undefined,
              "reminderAsterisk": undefined,
              "slidersArray": Array [
                "Passive",
                "Co-Regulation",
                "Approach",
                "Adaptive",
              ],
              "title": "Regulatory Capacities",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "mod_4b_active,mod_4b_self_regulation,mod_4b_avoid,mod_4b_maladaptive",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "mod_4b_active",
                "mod_4b_self_regulation",
                "mod_4b_avoid",
                "mod_4b_maladaptive",
              ],
              "reminder": undefined,
              "reminderAsterisk": undefined,
              "slidersArray": Array [
                "Active",
                "Self-Regulation",
                "Avoid",
                "Maladaptive",
              ],
              "title": "Regulatory Capacities",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <SessionHeader
            databaseNamesArray={
              Array [
                "mod_h_vestibular",
                "mod_h_proprioception",
                "mod_h_tactile",
                "mod_h_auditory",
                "mod_h_visual",
                "mod_h_intero",
              ]
            }
            headings={
              Array [
                "Vestibular",
                "Proprioception",
                "Tactile",
                "Auditory",
                "Visual",
                "Intero",
              ]
            }
            title="Modulation Capacities: Sensory Modulation to Self-Regulation"
          />,
          <div
            className="category-content"
          >
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_1_defensiveness",
                  "mod_1_over",
                  "mod_1_under",
                  "mod_1_variable",
                ]
              }
              reminder="* If selected, remember to make a hypothesized arousal chart"
              reminderAsterisk={
                Array [
                  null,
                  "*",
                  "*",
                  "*",
                  "*",
                ]
              }
              slidersArray={
                Array [
                  "Defensiveness",
                  "Over",
                  "Under",
                  "Variable",
                ]
              }
              title="Capacities"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_2_attention",
                  "mod_2_action",
                  "mod_2_affect",
                  "mod_2_autonomic",
                  "mod_2_arousability",
                ]
              }
              reminder={undefined}
              reminderAsterisk={
                Array [
                  null,
                  null,
                  null,
                  null,
                  "*",
                ]
              }
              slidersArray={
                Array [
                  "Attention",
                  "Action",
                  "Affect",
                  "Autonomic",
                  "Arousability",
                ]
              }
              title="Capacities"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_3_physiological",
                  "mod_3_sensory",
                  "mod_3_motor",
                  "mod_3_emotional",
                  "mod_3_cognitive",
                  "mod_3_social",
                  "mod_3_prosocial",
                ]
              }
              reminder={undefined}
              reminderAsterisk={undefined}
              slidersArray={
                Array [
                  "Physiological",
                  "Sensory",
                  "Motor",
                  "Emotional",
                  "Cognitive",
                  "Social",
                  "Prosocial",
                ]
              }
              title="Self-Regulation"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_4a_passive",
                  "mod_4a_co_regulation",
                  "mod_4a_approach",
                  "mod_4a_adaptive",
                ]
              }
              reminder={undefined}
              reminderAsterisk={undefined}
              slidersArray={
                Array [
                  "Passive",
                  "Co-Regulation",
                  "Approach",
                  "Adaptive",
                ]
              }
              title="Regulatory Capacities"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "mod_4b_active",
                  "mod_4b_self_regulation",
                  "mod_4b_avoid",
                  "mod_4b_maladaptive",
                ]
              }
              reminder={undefined}
              reminderAsterisk={undefined}
              slidersArray={
                Array [
                  "Active",
                  "Self-Regulation",
                  "Avoid",
                  "Maladaptive",
                ]
              }
              title="Regulatory Capacities"
            />
          </div>,
        ],
        "className": "Modulation",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "databaseNamesArray": Array [
              "mod_h_vestibular",
              "mod_h_proprioception",
              "mod_h_tactile",
              "mod_h_auditory",
              "mod_h_visual",
              "mod_h_intero",
            ],
            "headings": Array [
              "Vestibular",
              "Proprioception",
              "Tactile",
              "Auditory",
              "Visual",
              "Intero",
            ],
            "title": "Modulation Capacities: Sensory Modulation to Self-Regulation",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "mod_1_defensiveness",
                    "mod_1_over",
                    "mod_1_under",
                    "mod_1_variable",
                  ]
                }
                reminder="* If selected, remember to make a hypothesized arousal chart"
                reminderAsterisk={
                  Array [
                    null,
                    "*",
                    "*",
                    "*",
                    "*",
                  ]
                }
                slidersArray={
                  Array [
                    "Defensiveness",
                    "Over",
                    "Under",
                    "Variable",
                  ]
                }
                title="Capacities"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "mod_2_attention",
                    "mod_2_action",
                    "mod_2_affect",
                    "mod_2_autonomic",
                    "mod_2_arousability",
                  ]
                }
                reminder={undefined}
                reminderAsterisk={
                  Array [
                    null,
                    null,
                    null,
                    null,
                    "*",
                  ]
                }
                slidersArray={
                  Array [
                    "Attention",
                    "Action",
                    "Affect",
                    "Autonomic",
                    "Arousability",
                  ]
                }
                title="Capacities"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "mod_3_physiological",
                    "mod_3_sensory",
                    "mod_3_motor",
                    "mod_3_emotional",
                    "mod_3_cognitive",
                    "mod_3_social",
                    "mod_3_prosocial",
                  ]
                }
                reminder={undefined}
                reminderAsterisk={undefined}
                slidersArray={
                  Array [
                    "Physiological",
                    "Sensory",
                    "Motor",
                    "Emotional",
                    "Cognitive",
                    "Social",
                    "Prosocial",
                  ]
                }
                title="Self-Regulation"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "mod_4a_passive",
                    "mod_4a_co_regulation",
                    "mod_4a_approach",
                    "mod_4a_adaptive",
                  ]
                }
                reminder={undefined}
                reminderAsterisk={undefined}
                slidersArray={
                  Array [
                    "Passive",
                    "Co-Regulation",
                    "Approach",
                    "Adaptive",
                  ]
                }
                title="Regulatory Capacities"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "mod_4b_active",
                    "mod_4b_self_regulation",
                    "mod_4b_avoid",
                    "mod_4b_maladaptive",
                  ]
                }
                reminder={undefined}
                reminderAsterisk={undefined}
                slidersArray={
                  Array [
                    "Active",
                    "Self-Regulation",
                    "Avoid",
                    "Maladaptive",
                  ]
                }
                title="Regulatory Capacities"
              />,
            ],
            "className": "category-content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "mod_1_defensiveness,mod_1_over,mod_1_under,mod_1_variable",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "mod_1_defensiveness",
                  "mod_1_over",
                  "mod_1_under",
                  "mod_1_variable",
                ],
                "reminder": "* If selected, remember to make a hypothesized arousal chart",
                "reminderAsterisk": Array [
                  null,
                  "*",
                  "*",
                  "*",
                  "*",
                ],
                "slidersArray": Array [
                  "Defensiveness",
                  "Over",
                  "Under",
                  "Variable",
                ],
                "title": "Capacities",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "mod_2_attention,mod_2_action,mod_2_affect,mod_2_autonomic,mod_2_arousability",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "mod_2_attention",
                  "mod_2_action",
                  "mod_2_affect",
                  "mod_2_autonomic",
                  "mod_2_arousability",
                ],
                "reminder": undefined,
                "reminderAsterisk": Array [
                  null,
                  null,
                  null,
                  null,
                  "*",
                ],
                "slidersArray": Array [
                  "Attention",
                  "Action",
                  "Affect",
                  "Autonomic",
                  "Arousability",
                ],
                "title": "Capacities",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "mod_3_physiological,mod_3_sensory,mod_3_motor,mod_3_emotional,mod_3_cognitive,mod_3_social,mod_3_prosocial",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "mod_3_physiological",
                  "mod_3_sensory",
                  "mod_3_motor",
                  "mod_3_emotional",
                  "mod_3_cognitive",
                  "mod_3_social",
                  "mod_3_prosocial",
                ],
                "reminder": undefined,
                "reminderAsterisk": undefined,
                "slidersArray": Array [
                  "Physiological",
                  "Sensory",
                  "Motor",
                  "Emotional",
                  "Cognitive",
                  "Social",
                  "Prosocial",
                ],
                "title": "Self-Regulation",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "mod_4a_passive,mod_4a_co_regulation,mod_4a_approach,mod_4a_adaptive",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "mod_4a_passive",
                  "mod_4a_co_regulation",
                  "mod_4a_approach",
                  "mod_4a_adaptive",
                ],
                "reminder": undefined,
                "reminderAsterisk": undefined,
                "slidersArray": Array [
                  "Passive",
                  "Co-Regulation",
                  "Approach",
                  "Adaptive",
                ],
                "title": "Regulatory Capacities",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "mod_4b_active,mod_4b_self_regulation,mod_4b_avoid,mod_4b_maladaptive",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "mod_4b_active",
                  "mod_4b_self_regulation",
                  "mod_4b_avoid",
                  "mod_4b_maladaptive",
                ],
                "reminder": undefined,
                "reminderAsterisk": undefined,
                "slidersArray": Array [
                  "Active",
                  "Self-Regulation",
                  "Avoid",
                  "Maladaptive",
                ],
                "title": "Regulatory Capacities",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
